pipelines/rancher/deploy/ui-bundle-deploy.groovy

```diff
@@ -0,0 +1,79 @@
+import org.folio.Constants
+import org.folio.rest.model.OkapiTenant
+import org.folio.utilities.model.Project
+
+@Library('pipelines-shared-library@RANCHER-463') _
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '20')),
+    disableConcurrentBuilds(),
+    parameters([
+        jobsParameters.refreshParameters(),
+        jobsParameters.clusterName(),
+        jobsParameters.projectName(),
+        jobsParameters.uiBundleTag(),
+        jobsParameters.uiBundleBuild(),
+        jobsParameters.repository(),
+        jobsParameters.branch(),
+        jobsParameters.tenantId()
+    ])
+])
+
+OkapiTenant tenant = new OkapiTenant(id: params.tenant_id)
+
+Project project_model = new Project(
+    hash: params.ui_bundle_build ? common.getLastCommitHash(params.folio_repository, params.folio_branch) : '',
+    clusterName: params.rancher_cluster_name,
+    projectName: params.rancher_project_name,
+    configType: 'development',
+    domains: [ui   : common.generateDomain(params.rancher_cluster_name, params.rancher_project_name, tenant.getId(), Constants.CI_ROOT_DOMAIN),
+              okapi: common.generateDomain(params.rancher_cluster_name, params.rancher_project_name, 'okapi', Constants.CI_ROOT_DOMAIN),
+              edge : common.generateDomain(params.rancher_cluster_name, params.rancher_project_name, 'edge', Constants.CI_ROOT_DOMAIN)],
+)
+
+project_model.uiBundleTag = params.ui_bundle_build ? "${project_model.getClusterName()}-${project_model.getProjectName()}-${tenant.getId()}-${project_model.getHash().take(7)}" : params.ui_bundle_tag
+
+ansiColor("xterm") {
+    common.refreshBuidParameters(params.refresh_parameters)
+    node("jenkins-agent-java11") {
```

Comment:
Can we use here dynamic agents jobsParameters.agents(), ?

Reply:
Just to reduce parameters number for end users. Cause here it is not required to specify java version